src/frontend/komponenter/Fagsak/Vilkaar/VelgVilkaarene.tsx

```diff
@@ -0,0 +1,122 @@
+
+interface IProps {
+    fagsak: IFagsak;
+}
+
+const VelgVilkaarene: React.FunctionComponent<IProps> = ({ fagsak }) => {
+    const history = useHistory();
+    const [fyltvilkår, settFyltvilkår] = React.useState(true);
+    const [feilmelding, settFeilmelding] = React.useState('');
+    const [senderInn, settSenderInn] = React.useState(false);
+
+    const aktivBehandling = fagsak.behandlinger.find(
+        (behandling: IBehandling) => behandling.aktiv === true
+    );
+
+    const aktivVedtak = aktivBehandling
+        ? aktivBehandling.vedtakForBehandling.find(
+              (vedtak: IVedtakForBehandling) => vedtak.aktiv === true
+          )
+        : undefined;
+
+    React.useEffect(() => {
+        if (aktivVedtak) {
+            settFyltvilkår(aktivVedtak.resultat == VedtakResultat.INNVILGET);
+        }
+    }, []);
+
+    if (!aktivBehandling) {
+        return (
+            <div>
+                <Normaltekst>Ingen aktiv behandling</Normaltekst>
+            </div>
+        );
+    }
+
+    return (
+        <div>
+            {feilmelding !== '' && <AlertStripe type="feil">{feilmelding}</AlertStripe>}
+            <div className={'vilkårene'}>
+                <Systemtittel children={'Inngangsvilkår'} />
+
+                <br />
+                <RadioPanelGruppe
+                    name={'inngangsvilkår'}
+                    legend={'Inngangsvikår for barnetrygd'}
+                    radios={[
+                        { label: 'Inngangsvilkårene er oppfylt', value: 'oppfylt' },
+                        { label: 'Inngangsvilkårene er ikke oppfylt', value: 'ikkeoppfylt' },
+                    ]}
+                    checked={fyltvilkår ? 'oppfylt' : 'ikkeoppfylt'}
+                    onChange={(evt: {}, value: string) => {
+                        settFyltvilkår(value === 'oppfylt');
+                    }}
+                />
+
+                <br />
+                <div className={'fastsett__navigering'}>
+                    <Knapp
+                        type={'hoved'}
+                        onClick={() => {
+                            history.push(`/fagsak/opprett`);
+                        }}
+                        children={'Tilbake'}
+                    />
+                    <Knapp
+                        type={'hoved'}
+                        spinner={senderInn}
+                        onClick={() => {
+                            const result = fyltvilkår
+                                ? VedtakResultat.INNVILGET
+                                : VedtakResultat.AVSLÅTT;
+                            const nextTo =
+                                result == VedtakResultat.INNVILGET
+                                    ? `/fagsak/${fagsak.id}/behandle`
+                                    : `/fagsak/${fagsak.id}/vedtak`;
+                            if (aktivVedtak?.resultat !== result) {
+                                settSenderInn(true);
+                                axiosRequest<IFagsak>({
```

Comment:
Note to team: we should move this logic to the /api folder in the future. I was in a hurry for the previous MVP and forgot to move them afterwards 👎

Reply:
agree